src/test/java/framework/actions_apps/MobileBasePage.java

```diff
@@ -85,4 +85,32 @@ protected void copyText(MobileElement mobileElement) {
     protected void compare(String actualText, String expectedText) {
         Assert.assertEquals(actualText, expectedText);
     }
+
+    public void textDisplayed(MobileElement mobileElement) {
+        try {
+            mobileElement.isDisplayed();
+        }
+        catch (NoSuchElementException e) {
+            e.printStackTrace();
+            throw new NoSuchElementException("Unable to locate the Element using: " + mobileElement.toString());
+        }
+    }
+
+    public void enabled(MobileElement mobileElement) {
+        try {
+            mobileElement.isEnabled();
+        } catch (NoSuchElementException e) {
+            e.printStackTrace();
+            throw new NoSuchElementException("Unable to locate the Element using: " + mobileElement.toString());
+        }
+    }
+
+    public static void swipeVertical( int xPress, int yPress, int xMoveTo, int yMoveTo) throws Exception {
```

Comment:
Modify this method to make it more dynamic approach. single method that is capable of swiping right/left